src/object-properties.js

```diff
@@ -159,12 +160,13 @@ const properties = {
  * @property {boolean=} [useColorExpression=false] - Set to true to use color expression
  * @property {PaletteColor=} color - Color defined by index or hex code, needed if useColorExpression is false
  * @property {ColorExpression=} colorExpression - Color defined by expression, needed if useColorExpression is true
+ * @property {string=} [mode = 'none'] - Set to media library to show background image
  * @property {boolean=} [useImage=false] - Set to true to show background image
```

Comment:
wait, do we need both here? looks like useImage and mode does the same thing

Reply:
Well...the old component of use-background-image in the main panel(not styling one) is a switch that here it is defined by a boolean useImage. The new one is a dropdown that is called mode, and has two options to show/hide the image, so should keep both.